apps/docs/docs/getting_started/starter_tutorial/chatbot.mdx

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_position: 2
+---
+
+# Chatbot tutorial
+
+Once you've mastered basic [retrieval-augment generation](retrieval_augmented_generation) you may want to create an interface to chat with your data. You can do this step-by-step, but we recommend getting started quickly using `create-llama`.
+
+## Using create-llama
+
+`create-llama` is a powerful but easy to use command-line tool that generates a working, full-stack web application that allows you to chat with your data. You can learn more about it on [the `create-llama` README page](https://www.npmjs.com/package/create-llama).
```

Comment:
create-llama also supports adding agent tools (Wikipedia, Weather, Web search) to the chat. If tools are added AND the user selects custom data (create-llama calls that datasources), then a QueryEngineTool is added.